int64 trig
int64 echo
---
float64 reading1
float64 reading2